frontend/src/routes/(user)/cafe/+page.svelte

```diff
@@ -6,18 +6,20 @@
 </script>
 
 <div class="cafe__body max-h-screen min-h-screen flex flex-col">
+	<!-- Sticky Header -->
 	<div class="overflow-clip top-0 sticky z-10">
 		<img
-			src={roof}
-			class="object-cover -translate-y-10 w-full scale-75 scale-x-105"
-			alt="cafe and pastries roof"
-		/>
+	src={roof}
+	class="object-cover -translate-y-10 w-full scale-75 scale-x-105 min-h-[175px] sm:min-h-[175px] md:min-h-[200px] lg:min-h-[200px]"
```

Comment:
these ranges are redundant. when you add a breakpoint, it means from-breakpoint-onwards.

eg. `h-10 sm:h-20` would render h-10 for lower than the sm width breakpoint and h-20 for anything higher, including md and lg.

therefore, `lg:min-h-[200px]` is doing nothing here since it's the same as the one set at a lower breakpoint

[See docs here](https://tailwindcss.com/docs/responsive-design)